sys/src/9/amd64/mmu.c

```diff
@@ -646,9 +646,9 @@ vmap(uintptr_t pa, usize size)
 	uintptr_t va;
 	usize o, sz;
 
-	DBG("vmap(%#p, %lu) pc=%#p\n", pa, size, getcallerpc());
+	if(DBGFLG) print("vmap(%#p, %lu) pc=%#p\n", pa, size, getcallerpc());
```

Comment:
Why is this changed from DBG?  Isn't it doing the same thing?

Reply:
I was printing with DBG, it's defined as a print so it wasn't really being reserved for debugging. That's why I chose a different way.